pkg/rke2/rke2.go

```diff
@@ -86,7 +86,9 @@ func setup(clx *cli.Context, cfg Config) error {
 	agentImagesDir := filepath.Join(dataDir, "agent", "images")
 
 	managed.RegisterDriver(&etcd.ETCD{})
-
+	if clx.IsSet("node-external-ip") && (clx.IsSet("cloud-provider-config") || clx.IsSet("cloud-provider-name")) {
+		return fmt.Errorf("Can't set node-external-ip while using cloud provider")
```

Comment:
No need to use the format error creation method here since we're not formatting anything. Also, we don't want to capitalize error strings. Link below for reference.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

```suggestion
		return errors.New("can't set node-external-ip while using cloud provider")
```